apps/jobs/constants/values.py

```diff
@@ -1,11 +1,21 @@
-"""This file contains constant which are also used in other parts of the jobs API code, 
+"""This file contains constant which are also used in other parts of the jobs API code,
 It helps you to declare the variable with their value here, and use them anywhere else"""
 
+STATUS_CHOICES = (
+    ("under-reviewed", "Under-Reviewed"),
+    ("shortlisted", "Shortlisted"),
+    ("accepted", "Accepted"),
+    ("rejected", "Rejected"),
+    ("on-hold", "On-Hold"),
+)
+
+GENDER = (("male", "Male"), ("female", "Female"), ("other", "Other"))
```

Comment:
I dont think so you have to keep all the case sensitive values.

while validating it or consuming it you can convert according to your use case.